common/lib/container-definitions/src/loader.ts

```diff
@@ -252,20 +252,37 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 
     /**
      * Boolean indicating whether the container is currently connected or not
+     * @deprecated - 0.58, This API will be removed in 0.60
+     * Use `connectionState` instead
+     * See https://github.com/microsoft/FluidFramework/issues/9167 for context
      */
     readonly connected: boolean;
 
+    /**
+     * Try to connect the container to the delta stream
+     */
+    connect?(): void;
```

Comment:
Why not add new APIs on class, propagate and wait some time, and then add them here, but not make them optional?
Alternatively, we can make them non-optional from start - I do not see big problem here. As long as we are adding implementation in parallel, package bump would be noop in the future.

Reply:
Yeah, that makes sense. I need to create a follow up PR to add `connect()` and `disconnect()` to `IFluidContainer` anyway, so we could just add in the functions to both `IContainer` and `IFluidContainer` in one PR.